packages/cli/src/zosfiles/edit/Edit.options.ts

```diff
@@ -0,0 +1,42 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandOptionDefinition } from "@zowe/imperative";
+
+import i18nTypings from "../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../-strings-/en").default as typeof i18nTypings).EDIT.OPTIONS;
+
+/**
+ * Object containing all options to be used by the View API
+ */
+export const EditOptions: { [key: string]: ICommandOptionDefinition } = {
+
+    /**
+     * The option to set a default editor
+     * @type {ICommandOptionDefinition}
+     */
+    editor: {
+        name: "editor",
+        aliases: ["ed"],
+        description: strings.EDITOR,
+        type: "string",
+        required: false
+    },
+    extension: {
+        name: "extension",
+        aliases: ["e"],
```

Comment:
The alias `--e` is convenient but also ambiguous since users might expect `--e` to be an alias for `--editor`.

Would it make sense to use the alias `--ext` here instead?

Reply:
![image](https://user-images.githubusercontent.com/112635587/230424583-7365c476-c8f2-42f2-a1ab-5cbf0f1e884f.png)

Accidentally attaching the largest picture ever but I went with --e because other commands implement this. If changing to --ext, should i edit all other instances of --e->--ext to be consistent?

Reply:
Sorry, didn't realize that other commands implement this as `-e`. To maintain backwards compatibility and consistency, I agree with leaving it as-is 👍